configuration/stub-frontend-fed-config/ab_test.yml

```diff
@@ -9,31 +9,45 @@
 #         - name: 'no_logo'
 #           percent: 25
 #         - name: 'no_logo_privacy'
+#           percent: 0
+#   - idp_ordering:
+#       alternatives:
+#         - name: 'alphabetical'
+#           percent: 75
+#         - name: 'popularity'
+#           percent: 25
+#
+# Experiments should follow the above structure.
+# While a test is running, only the percentages should change. Only use whole numbers.
+# To make all users see the winning alternative, all the other alternatives should be removed, e.g.
+#
+#  experiments:
+#   - about_companies:
+#       alternatives:
+#         - name: 'no_logo'
+#           percent: 100
 #
-# Only the percentages should be changed, and they must be whole numbers, no decimals.
 # The value reported to Piwik will be the experiment name followed by the alternative name, e.g. "about_companies_with_logo"
+# Only the current experiment should have multiple alternatives so that Piwik reports just that experiment.
 # The first alternative will be used as the default for any issues.
-#
 # This file must exist and must contain at least:
 # experiments:
 #
 
 experiments:
-  - select_documents_v2:
-      alternatives:
-        - name: 'control'
-          percent: 100
-  - about_companies:
+  - short_hub_2019_q3-preview:
       alternatives:
-        - name: 'with_logo'
+        - name: 'control_a'
           percent: 100
-        - name: 'no_logo'
-          percent: 0
-        - name: 'no_logo_privacy'
+        - name: 'variant_c_2_idp_short_hub'
           percent: 0
-  - loa1_perf_management:
+  - short_hub_2019_q3:
       alternatives:
-        - name: 'control'
-          percent: 100
-        - name: 'variant'
+        - name: 'control_a'
           percent: 0
+        - name: 'variant_c_2_idp_short_hub'
```

Comment:
Short hub experiment is going away so do we need this for local hub? It's probably better to just make it run with control.
My concern is that this file will need to be updated every time we have an AB test, otherwise local running may break. It would be better to make it independent of AB tests or use the AB test config file from verify-frontend instead of replicating it. I think ida-hub-acceptance tests works that way.

Reply:
I can investigate this...  I think to meet the intentions of this repo which is to allow anyone to spin up the hub it has its own version of verify-hub-frontend-federation-config which is also where the files relating to the AB test normally live.  Let me do a quick investigation over lunchtime.

Reply:
Updated this to point to the one in verify-frontend repository.  Also updated the file to match what is currently in production to stop it breaking while we complete the AB test tear down work.  (See PR: https://github.com/alphagov/verify-frontend/pull/904)